requirements/edx/base.txt

```diff
@@ -380,6 +380,8 @@ djangorestframework==3.12.4
     #   super-csv
 djangorestframework-xml==2.0.0
     # via edx-enterprise
+docker==5.0.0
+    # via py2neo
```

Comment:
I hate hate hate that py2neo lists `docker` as a base requirement, but short of forking or refusing to upgrade, I'm not sure how we could fix this. I was tempted to try to submit a patch to py2neo, but didn't want to go down that rabbit hole right now.

Reply:
Yipes! That's some bloat. Oh well....